src/config.cpp

```diff
@@ -155,7 +155,7 @@ Config& Config::parse(int argc, char** argv) {
                 cxxopts::value<bool>()->default_value("false"),
                 "METRICS")
             ("metrics_list",
-                "List of metrics families to be enabled. If empty the default metrics are enabled: ovms_requests_success, ovms_requests_fail, ovms_request_time_us, ovms_streams, ovms_inference_time_us, ovms_wait_for_infer_req_time_us. Additional families: ovms_infer_req_queue_size, ovms_infer_req_active.",
+                "Comma separated list of metrics. If unset, there is assumed a default list. When set, only the listed metrics will be enabled. Default metrics: ovms_requests_success, ovms_requests_fail, ovms_request_time_us, ovms_streams, ovms_inference_time_us, ovms_wait_for_infer_req_time_us; Additional metrics: ovms_infer_req_queue_size, ovms_infer_req_active.",
```

Comment:
Don't we want to have by the default metrics used in HPA in K8S? I assume for those we require :
ovms_current_requests", "ovms_infer_req_active"?
@dkalinowski / @dtrawins

Reply:
We need  ovms_current_requests + ovms_streams for Autoscalling.